polkadot/xcm/xcm-simulator/fuzzer/src/parachain.rs

```diff
@@ -60,9 +84,9 @@ impl frame_system::Config for Runtime {
 	type RuntimeCall = RuntimeCall;
 	type Nonce = u64;
 	type Hash = H256;
-	type Hashing = ::sp_runtime::traits::BlakeTwo256;
+	type Hashing = BlakeTwo256;
 	type AccountId = AccountId;
-	type Lookup = IdentityLookup<Self::AccountId>;
+	type Lookup = AccountIdLookup<AccountId, ()>;
```

Comment:
Should be possibly to only write:  
```rust
	type AccountId = AccountId;
	type Lookup = AccountIdLookup<AccountId, ()>;
	type Block = Block;
	type AccountData = pallet_balances::AccountData<Balance>;
```
Since the others get pulled in from the `derive_impl`. Probably need to set `BlockNumber` to `u64`.

Reply:
Good catch, done